packages/react-components/src/components/ProgressButton/ProgressButton.jsx

```diff
@@ -9,7 +9,7 @@ import classNames from 'classnames';
 
 class ProgressButton extends React.Component {
   UNSAFE_componentWillMount() {
-    this.id = uniqueId();
+    this.id = uniqueId('continueButton-');
```

Comment:
Why is this changing in some places and not in others?

Reply:
Most places we used the prefix and in two files we didnt which was arbitrary and was making the id read 1-continueButton. So I adjusted it so that all files would use the prefix option when using uniqueId to keep the methodology consistent

Reply:
This is the kind of thing that can cause unintended side-effects. I'd also like to avoid it to keep the scope of this PR limited to removing lodash, rather than refactoring things that result in DOM changes.

[Here's an example of how this change could cause more work to be done](https://github.com/department-of-veterans-affairs/vets-website/blob/94b21a0f8db202254d94347b6fcae5198debc566/src/applications/health-care-questionnaire/list/tests/e2e/04.questionnaire.happy.path.cypress.spec.js#L31). Any CSS or JS could be targeting an id with the pattern `#N-continueButton`, and we would break those styles/code/tests.